src/formats/fasta.rs

```diff
@@ -0,0 +1,84 @@
+use std::io::Write;
+
+use memchr::memchr;
+
+use crate::buffer::RecBuffer;
+use crate::seq::Sequence;
+use crate::util::{memchr_both, strip_whitespace, ParseError, ParseErrorType};
+
+#[derive(Debug)]
+pub struct FASTA<'a> {
```

Comment:
Rust convention is to not uppercase struct/enums even if they are in theory, so FASTA -> Fasta